openshift_cli_installer/cli.py

```diff
@@ -199,6 +199,38 @@
 """,
     type=click.Path(exists=True),
 )
+@click.option(
+    "--azure-tenant-id",
+    help="""
+\b
+Azure's service principal tenant ID.
+""",
+    type=str,
+)
+@click.option(
+    "--azure-client-id",
+    help="""
+\b
+Azure's service principal client ID.
+""",
+    type=str,
+)
+@click.option(
+    "--azure-client-secret",
```

Comment:
Can we get default values for AZURE taken from `os.environ`, like we do for AWS values?

Reply:
done